### ISSUE TITLE
Check luci-interpreter's dependency on configure stage

### ISSUE THREAD

**Issue by cgbahk** [Samsung/ONE#2811]
cc @s-barannikov 

_This is a suggestion by user(me) experience ;)_

While configuring to use luci-interpreter library externally, I encountered that luci-interpreter was trying to be built even without its dependency: kernel libarary. It eventually occurs link error as it couldn't find kernel library.
```
No gemm/eigen
-> No luci interpreter kernel build
-> But luci interpreter is meant to be linked to kernel library anyway
-> link error
```

It might be better if we make this error, warning, or skip building,  earlier maybe at configure stage
(up to maintainer for the detail :smile:)

**Comment by s-barannikov** [comment]
@cgbahk 

Sorry for having you to debug the link error, and thanks for suggestion!
I would return an error if some of the dependencies could not be found, but I can see that other components usually return silently. Is it OK if I don't follow this practice?

**Comment by cgbahk** [comment]
> but I can see that other components usually return silently

I understand we cannot manage this for all :sob: I just report this as I encountered this case.

'return silently' (from my understanding, don't build) luci-interpreter when no (recursive) dependency case is natural. But IMHO building luci-interpreter without required dependency is a bit strange (like occurs link error in this case)

> Is it OK if I don't follow this practice?

So it's really up to you :) Actually I'm okay with current form. If there's a bit of improvement, that would be even better.

**Comment by cgbahk** [comment]
Resolved by #2840, thanks :+1:
